tests/test_l293d.py

```diff
@@ -84,6 +84,7 @@ def test_pin_numbering_lock(self):
                    'if motors already exist. Set this at '
                    'the start of your script.')
         m1.remove()
+        d.Config.pin_numbering = 'BOARD'
```

Comment:
I've had to reset this value in the test because I couldn't get `reload` to set the Config back to default.